packages/web/components/complex/assets-page-v1.tsx

```diff
@@ -268,14 +268,6 @@ const PoolCardsDisplayer: FunctionComponent<{ poolIds: string[] }> = observer(
 
         const apr =
           poolBonding.highestBondDuration?.aggregateApr ?? new RatePretty(0);
-
-        if (
-          !pool ||
-          (pool.type === "concentrated" && !flags.concentratedLiquidity)
```

Comment:
Should prob still keep the check for pool. Just remove the statement for the CL flag

Reply:
I have reverted and only removed the flag @jonator can you try rebuilding it.

Reply:
@jonator check